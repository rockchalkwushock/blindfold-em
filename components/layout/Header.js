import { styles } from '../../lib'

const Header = ({ children }) => (
  <header>
    {children}
    <style jsx>{`
      header {
        align-items: center;
        background-color: ${styles.colors.lowLight};
        display: flex;
        flex-direction: column;
        font-family: ${styles.fonts.permMarker};
        font-size: 2em;
        height: 20vh;
        justify-content: center;
        letter-spacing: 0.7em;
        padding: 1em;
        text-align: center;
      }
    `}</style>
  </header>
)

export default Header
