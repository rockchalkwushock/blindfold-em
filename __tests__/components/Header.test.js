import React from 'react'
import { mount } from 'enzyme'

import Header from '../../components/layout/Header'

const Test = () => (
  <Header>
    <div>1</div>
    <div>2</div>
    <div>3</div>
  </Header>
)

describe('Component: <Header />', () => {
  test('should mount without exploding', () => {
    const tree = mount(<Test />)
    expect(tree).toMatchSnapshot()
  })
})
