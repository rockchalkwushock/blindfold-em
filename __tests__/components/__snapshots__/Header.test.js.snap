// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Header /> should mount without exploding 1`] = `
<Test>
  <Header>
    <header
      className="jsx-2290366526"
    >
      <div>
        1
      </div>
      <div>
        2
      </div>
      <div>
        3
      </div>
      <JSXStyle
        css="header.jsx-2290366526{-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;background-color:lightcoral;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;height:20vh;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;padding:1em;text-align:center;}"
        styleId="2290366526"
      />
    </header>
  </Header>
</Test>
`;
